core/color.h

```diff
@@ -101,6 +101,34 @@ struct Color {
 		return res;
 	}
 
+	_FORCE_INLINE_ Color darken(float amt) {
```

Comment:
The convention in Godot is to prefix non-mutable parameters with `p_`, so it should be `p_amount`.

Reply:
This one should be `void`, it fails to compile now.

Reply:
You're right, it should, however unusually this doesn't even error the build for me.
> [100%] scons: done building targets.

Reply:
I guess it depends on how strict the compiler is. VS 2015 seems unhappy with it :)